helpers/formValidation.tsx

```diff
@@ -37,6 +37,15 @@ const lessonSchema = Yup.object({
   chatUrl: Yup.string()
 })
 
+const moduleSchema = Yup.object({
```

Comment:
`moduleSchemaValidation` to match the file naming pattern of `name+vValidation`?

Reply:
yeah good idea to keep things consistent.